.github/workflows/scheduled.yml

```
@@ -0,0 +1,57 @@
+name: Scheduled jobs
+
+on: push
+
+jobs:
+  check-broken-links:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+        with:
+          token: ${{ secrets.DEPLOYMENT_PERSONAL_ACCESS_TOKEN }}
+          submodules: true
+
+      - name: Check for broken links
+        id: lychee
+        uses: lycheeverse/lychee-action@v1.5.4
+        with:
+          args: --verbose ./docs/**/*.md
+          jobSummary: true
+        env:
+          GITHUB_TOKEN: ${{secrets.GITHUB_TOKEN}}
+
+      - name: Create issue
+        if: steps.lychee.outputs.exit_code != 0
```

[This example uses a different syntax](https://github.com/lycheeverse/lychee-action#usage) for testing the exit code:

```suggestion
        if: env.lychee_exit_code != 0
```

This must have changed at some point, since before it was working on codacy/docs. But now I can see [some runs of the workflow with errors](https://github.com/codacy/docs/actions/runs/3983598407/jobs/6829055828#step:3:47) that failed to create an issue! I'll open a pull request on codacy/docs to make this change there as well.

---

Well, I just found out that the change happened 3 months ago:

https://github.com/lycheeverse/lychee-action/commit/2fa80ee45a2e8110e4931078749158b45207e9b7

---

Fixed! 👍
A [new issue](https://github.com/codacy/pulse-user-docs/issues/186) was created.